documentation/raw_data_import_process.md

```diff
@@ -0,0 +1,84 @@
+# Raw Data Import Process
+
+## Purpose
+The stakeholder will need to import a raw dataset into the database once a year. 
+A script is useful to help automate this task.
+The stakeholder is able to use a command line to initiate the script.
+
+## Overview
+- Use a Python script to clean the original `Asylum Case Data` and convert to .csv 
+- Use AWS implementation for bulk .csv ingestion to import cleaned data to DynamoDB
+
+## Import Process
+### Prerequisites
+1. IAM User with AWS permissions
+   - CloudFormation
+   - DynamoDB 
+   - Lambda
+   - S3 
+2. AWS CLI Installed _(Latest Version)_
+      - aws_access_key_id
+      - aws_secret_access_key
+3. Python3 Installed with Pandas Library _(Latest Versions)_
+
+### Requirements
+- [COW2021001887-I589Data.xlsx](https://bloomtech.notion.site/HRF-Asylum-Report-Generator-412062ca7640457682c33295b21a25c3#:~:text=FE%3A%20React%20(JS)-,Resources,-Asylum%20decision%20data) _(Resources Section)_
+- [AsylumCaseXlsxToCleanCsv.py](/configurations/AsylumCaseXlsxToCleanCsv.py)
+- [CSVToDynamoDB.template.json](/configurations/CSVToDynamo.template.json)
+
+### Importing
+#### Deploy Stack 
+1. Open CLI in the root of this project
+2. CLI command for us-west-2 `aws cloudformation create-stack --region us-west-2 --stack-name asylum-rg-be-a --template-body file://configurations/CSVToDynamo.template.json --capabilities CAPABILITY_IAM`
```

Comment:
May want to add a verb before CLI. Ie. "Run the CLI command `aws cloudformation .... `